tests/reading-tests.md

```diff
@@ -38,7 +38,7 @@ They all differ in testing philosophy, strengths, responsibilities, and context.
 This curriculum focuses on Jest additionally because:
 
 - It is widely adopted in modern JavaScript projects
-- It is built into Replit.com
+- It is built into [CodeSandbox](https://codesandbox.io)
```

Comment:
acknowledge that we jumped to a completely different environment, but still tie this back to things we've seen (the reader _could_ implement this in replit if they so desired)
```suggestion
- It is built into several online editors, including [Replit.com](https://replit.com), which we have used before, as well as [CodeSandbox](https://codesandbox.io), which we'll explore a bit in this lesson.
```